### PR TITLE
Create s3backup.yaml

### PR DESCRIPTION
**Pull request by dnnrly**

### CODE REVIEW

File: _data/projects/s3backup.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (ritwik12) [inline _data/projects/s3backup.yaml]: @dnnrly File extension should be .yml instead of .yaml

**Review by ritwik12 — DISMISSED** [review]

**Comment by ritwik12** [comment]
@dnnrly I can't find any open issues with label `help wanted`, this projects looks more like a personal one. Can you please add few issues for others to contribute.

**Comment by dnnrly** [comment]
Ah yes, I got it feature complete and ready for public consumption. Just forgot to add the extra bits. 😓 

Issues are added now.

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @dnnrly!

**Pull request merged**